### PR TITLE
Use ProfileWithInlinePolicies for controls in profiles

### PR DESCRIPTION
**Pull request by jan-cerny**
In this format, the items of policies lists are tuples that consists
of a policy ID and control IDs separated by a colon.

### CODE REVIEW

File: rhel8/profiles/abcd-low-inline.profile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+documentation_complete: true
+title: ABCD Low for Red Hat Enterprise Linux 8
+description: >-
+  This profile contains configuration checks that align to
+  the ABCD benchmark.
+selections:
+  - security_patches_up_to_date
+  - abcd:R1
+  - abcd:R2
+  - abcd:R3
```
Comment (yuumasato) [inline rhel8/profiles/abcd-low-inline.profile:10]: I don't have a strong opinion, but I have slight preference on this one.

Comment (matejak) [reply rhel8/profiles/abcd-low-inline.profile:10]: What I like about it is that it allows to keep overrides of unwanted selections close to the control that introduces them, and it gives user greater freedom with position of security control selections in the file, especially if they come from more than one policy. Do we like the colon as a delimiter though?

Comment (yuumasato) [reply rhel8/profiles/abcd-low-inline.profile:10]: The colon looks fine for now.

**Comment by matejak** [comment]
We like prefer the "Inline" to "Separated", and this PR doesn't have any open issues, so I advise to merge it.

**Pull request merged**